prow/flagutil/github.go

```diff
@@ -66,6 +68,10 @@ func (o *GitHubOptions) Validate(dryRun bool) error {
 		}
 	}
 
+	if _, err := url.ParseRequestURI(o.GitEndpoint); err != nil {
+		return fmt.Errorf("invalid -git-endpoint URI: %q", o.GitEndpoint)
```

Comment:
We tend to use `--git-endpoint` for flags everywhere

Reply:
oh you probably mean --github-endpoint. Want me to search-replace all `-git-endpoint` with `--git-endpoint`?

Reply:
No I mean as opposed to `-git-endpoint` with only one hypen